p2p/src/sync/mod.rs

```diff
@@ -355,30 +355,29 @@ where
     ///
     /// The node is considered fully synced (its initial block download is done) if all its peers
```

Comment:
As we've discussed this is totally wrong and has to be changed in some subsequent PR.